openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/ProjectRequestsOperationImpl.java

```diff
@@ -119,19 +121,23 @@ public Status list() {
 
   @Override
   public Status list(Integer limitVal, String continueVal) {
+    return list(new ListOptionsBuilder().withLimit(Long.parseLong(limitVal.toString())).withContinue(continueVal).build());
+  }
+
+  @Override
+  public Status list(ListOptions listOptions) {
     try {
       URL requestUrl = getNamespacedUrl();
-      if(limitVal != null) {
-        requestUrl = new URL(URLUtils.join(requestUrl.toString(), "?limit=" + limitVal.toString()));
-      } else if(continueVal != null) {
-        requestUrl = new URL(URLUtils.join(requestUrl.toString(), "?limit=" + limitVal.toString() + "&continue=" + continueVal));
+      if(listOptions.getLimit() != null) {
+        requestUrl = new URL(URLUtils.join(requestUrl.toString(), "?limit=" + listOptions.getLimit().toString()));
+      } else if(listOptions.getContinue() != null) {
```

Comment:
This doesn't look good as getLimit will return null.
Why not use the same Url builder you used in BaseOperation